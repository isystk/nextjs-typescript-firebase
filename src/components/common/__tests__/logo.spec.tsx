import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Logo from '../logo'

test('Logo', () => {
  const wrapper = shallow(<Logo />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
