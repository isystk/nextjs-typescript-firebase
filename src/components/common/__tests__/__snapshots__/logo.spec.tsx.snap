// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo 1`] = `
<div
  className="header-logo"
>
  <Link
    href="https://localhost/"
  >
    <styled.h1>
      Isystk's Frontend Sample
    </styled.h1>
  </Link>
</div>
`;
